include/cute/util/sycl_vec.hpp

```diff
@@ -52,6 +52,7 @@ using uchar2 = vector_t<uchar, 2>;
 using uchar4 = vector_t<uchar, 4>;
 using uchar8 = vector_t<uchar, 8>;
 using uchar16 = vector_t<uchar, 16>;
+using uchar64 = vector_t<uchar, 64>;
```

Comment:
```suggestion
using uchar32 = vector_t<uchar, 32>;
using uchar64 = vector_t<uchar, 64>;
```
Just for completeness :smile: